### PR TITLE
Iz gawb 1232 remember my view when i export

### PR DESCRIPTION
**Pull request by zarsky-broad**
- [x] **Submitter**: Include the JIRA issue number in the PR description
- [x] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [x] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * Add notes on what you've tested
- [x] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [x] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [x] **Submitter**: If you're adding new libraries, sign us up to security updates for them
- [x] Tell the tech lead (TL) that the PR exists if they wants to look at it
- [x] Anoint a lead reviewer (LR). **Assign PR to LR**
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [x] **TL** sign off
- [x] **LR** sign off
- [x] **Product Owner** sign off
- [x] **Assign to submitter** to finalize
- [x] **Submitter**: Verify all tests go green, including CI tests
- [x] **Submitter**: Squash commits and merge to develop
- [x] **Submitter**: Delete branch after merge
- [x] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [x] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

File: src/cljs/org/broadinstitute/firecloud_ui/page/workspace/data/tab.cljs [hunk 1]
```diff
@@ -13,7 +13,8 @@
     [org.broadinstitute.firecloud-ui.endpoints :as endpoints]
     [org.broadinstitute.firecloud-ui.page.workspace.data.copy-data-workspaces :as copy-data-workspaces]
     [org.broadinstitute.firecloud-ui.page.workspace.data.import-data :as import-data]
-    [org.broadinstitute.firecloud-ui.utils :as utils :refer [access-token]]))
+    [org.broadinstitute.firecloud-ui.utils :as utils :refer [access-token]]
+    [org.broadinstitute.firecloud-ui.persistence :as persistence]))
```
Comment (dmohs) [inline src/cljs/org/broadinstitute/firecloud_ui/page/workspace/data/tab.cljs]: Alphabetical please so we don't have to think about it.

File: src/cljs/org/broadinstitute/firecloud_ui/page/workspace/data/tab.cljs [hunk 2]
```diff
@@ -82,7 +83,15 @@
                             [:a {:style {:textDecoration "none" :margin "7px 0 0 1em"}
                                  :href (str (config/api-url-root) "/cookie-authed/workspaces/"
                                             (:namespace workspace-id) "/"
-                                            (:name workspace-id) "/entities/" selected-entity-type "/tsv")
+                                            (:name workspace-id) "/entities/" selected-entity-type "/tsv"
+                                            "?attributeList="
+                                            (->> (persistence/try-restore
+                                                  {:key (str (common/workspace-id->string workspace-id) ":data:" selected-entity-type)
```
Comment (dmohs) [inline src/cljs/org/broadinstitute/firecloud_ui/page/workspace/data/tab.cljs:89]: Nit: Does this line respect 100 column limit?

File: src/cljs/org/broadinstitute/firecloud_ui/page/workspaces_list.cljs [hunk 3]
```diff
@@ -101,7 +101,9 @@
    (fn []
      (persistence/try-restore
       {:key persistence-key
-       :initial (fn [] {:selected-types (->> access-types (map (juxt identity (constantly true))) (into {}))})}))
+       :initial (fn [] {:selected-types (->> access-types
```
Comment (dmohs) [inline src/cljs/org/broadinstitute/firecloud_ui/page/workspaces_list.cljs]:
``` clojure
(->> access-types (map (juxt identity (constantly true))) (into {}))
```

can be re-written as:

``` clojure
(reduce (fn [m x] (assoc m x true)) {} access-types)
```

which seems clearer to me.

**Review by dmohs — APPROVED** [review]

**Comment by dvoet** [comment]
👍

**Comment by dvoet** [comment]
I reviewed with Tiffany and she approved

**Pull request merged**